test/relationships/int.spec.ts

```diff
@@ -565,7 +586,7 @@ describe('Relationships', () => {
   })
 
   describe('Polymorphic Relationships', () => {
-    it('should allow REST querying on polymorphic relationships', async () => {
+    it('should allow Local API querying on polymorphic relationships', async () => {
```

Comment:
This test case is still using `client` to do the actual querying so the change here isn't accurate.